test/benchmark/OsmReporter.mjs

```diff
@@ -0,0 +1,23 @@
+// my-awesome-reporter.js
```

Comment:
This is an example of a custom reporter, which we can edit how we want in case we have different needs for formatting of the output from the benchmarks. It's not used as of now.